src/Roslyn.Diagnostics.Analyzers/CSharp/CSharpCreateTestAccessorFixer.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Roslyn.Diagnostics.Analyzers;
+
+namespace Roslyn.Diagnostics.CSharp.Analyzers
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
+    public class CSharpCreateTestAccessorFixer : CreateTestAccessorFixer
+    {
+        protected override SyntaxNode GetTypeDeclarationForNode(SyntaxNode reportedNode)
```

Comment:
❓ Do you have any "policy" on when to use method body or method arrow expression?

Reply:
We do not